Tests/GauntletTests/Operators/EqualWithAccuracyOperatorsTests.swift

```diff
@@ -133,7 +134,8 @@ class EqualWithAccuracyAssertsTestCase: XCTestCase {
         // Then
         Assert(that: assertion)
             .didFail(expectedName: "isNotEqualTo(, accuracy)", expectedLine: expectedLine)
-            .isEqualTo(.message("4 is equal to the expected value 3. Accuracy: 2"))
+            .isMessage()
+            .isEqualTo("4 is equal to the expected value 3. Accuracy: 2")
```

Comment:
Something this change made me realize: The Assertion operators are past tense (didFail, didPass), while all other operators are present tense. This really feels like it clashes. I originally named those operators with past tense because it read better to me than `isPass` or `isFail`, but that feels like it falls apart in context. Do you think I should update the Assertion operators to use `is` to be consistent with the rest of the API?

Reply:
I think it's okay as-is because `didFail` is a verb and the others are not. Otherwise it would need to be `isFailure`, which seems worse.

Reply:
Were it to be `isFailure`, I would then think of `hasMessage`, and `equalTo` so that it read more like a sentence.

Reply:
The verb distinction is a good one. On `hasMessage` and `equalTo`, I want to have a naming pattern that is consistent such that they all fit reasonably well with each other. `equalTo` would read nicely when after a `hasMessage` but wouldn't read as nicely after `Assert(that: 5).equalTo(...)`.